File: JAVA/Sivitas.java
class Sivitas extends Human
{
    private String asalUniv;
    private String emailEdu;

    Sivitas()
    {
        this.asalUniv = "";
        this.emailEdu = "";
    }

    Sivitas(String a, String b, String c, String d, String e)
    {
        setNik(a);
        setNama(b);
        setJk(c);
        this.asalUniv = d;
        this.emailEdu = e;
    }

    void setAsalUniv(String a)
    {
        this.asalUniv = a;
    }
    void setEmailEdu(String a)
    {
        this.emailEdu = a;
    }

    String getAsalUniv()
    {
        return this.asalUniv;
    }
    String getEmailEdu()
    {
        return this.emailEdu;
    }
};

File: PHP/Sivitas.php
<?php
include "Human.php";
class Sivitas extends Human
{
    private $asalUniv = "";
    private $emailEdu = "";
    function __construct($n, $na, $j, $u, $e)
    {
        $this->setNik($n);
        $this->setNama($na);
        $this->setJk($j);
        $this->asalUniv = $u;
        $this->emailEdu = $e;
    }
    function setAsalUniv($a)
    {
        $this->asalUniv = $a;
    }
    function setEmailEdu($a)
    {
        $this->emailEdu = $a;
    }

    function getAsalUniv()
    {
        return $this->asalUniv;
    }
    function getEmailEdu()
    {
        return $this->emailEdu;
    }
    function __destruct(){
        
    }
}
?>
